target/linux/bcm5862x/image/Makefile

```diff
@@ -0,0 +1,52 @@
+# SPDX-License-Identifier: GPL-2.0-only
+#
+# Copyright (C) 2021 Matthew Hagan <mnhagan88@gmail.com>
+
+include $(TOPDIR)/rules.mk
+include $(INCLUDE_DIR)/image.mk
+
+define Build/MerakiNAND-old
+	-$(STAGING_DIR_HOST)/bin/mkmerakifw-old \
+		-B mx64 -s \
+		-i $@ \
+		-o $@.new
+	mv $@.new $@
+endef
+
+define Device/Default
+  DEVICE_DTS := bcm958*-$(lastword $(subst _, ,$(1)))
```

Comment:
The asterisk here would be an interesting option for other targets _if_ it works reliably.
I do remember, but I think there was a reason why we don't use this ...

Reply:
It seemed to be working fine but I have modified to the style of commit [9c7025b](https://github.com/openwrt/openwrt/commit/9c7025b7794ae06774632499547f78d827d13499) which is a bit more precise.